scipy/stats/_continuous_distns.py

```diff
@@ -4002,8 +4003,10 @@ def _cdf(self, x, alpha, beta):
                 data_out[data_mask] = np.array([levy_stable._cdf_single_value_zolotarev(_x, _alpha, _beta)
                             for _x, _alpha, _beta in data_subset]).reshape(len(data_subset), 1)
             else:
-                warnings.warn('Cumulative density calculations experimental for FFT method.' +
-                              ' Use zolatarev method instead.', RuntimeWarning)
+                warnings.warn(u'FFT method is considered experimental for ' +
```

Comment:
Ah, this is the test failure - the changed message probably bypasses the existing warning filter